swagger/src/test/scala/org/http4s/rho/swagger/TypeBuilderSpec.scala

```diff
@@ -291,6 +297,17 @@ class TypeBuilderSpec extends Specification {
           prop.`type` must_== "string"
       }
     }
+
+    "Build a model for sealed traits" in {
+      val ms = modelOf[Sealed]
+      ms.foreach(_.toJModel) // Testing that there are no exceptions
+      ms.size must_== 3
+      val Some(seal: models.ModelImpl) = ms.find(_.id2 == "Sealed")
+      seal.discriminator must_== Some("foobar")
+      val Some(foo: models.ComposedModel) = ms.find(_.id2 == "FooSealed")
+      val Some(fooRef) = foo.allOf.collectFirst({case ref: models.RefModel => ref})
```

Comment:
Don't worry about this `({` if you don't end up making another commit, but I'd like to get rid of it if you do.